lib/rules/no-unused-prop-types.js

```diff
@@ -17,7 +17,7 @@ var annotations = require('../util/annotations');
 // Constants
 // ------------------------------------------------------------------------------
 
-var DIRECT_PROPS_REGEX = /^props\s*(\.|\[)/;
+var DIRECT_PROPS_REGEX = /^(props|nextProps)\s*(\.|\[)/;
```

Comment:
this seems to suggest that there will be places that `props` and `nextProps` are interchangeable - that doesn't seem right to me.